inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
```suggestion
<?php
declare(strict_types=1);
```

Let's start adding the strict typing in our new classes from now on